index.html

```diff
@@ -27,4 +24,6 @@ <h4>Great Sightings</h4>
   </div>
 </body>
 
-</html>
\ No newline at end of file
+</html>
+<script src="data.js"></script>
+<script src="index.js"></script>
```

Comment:
Good job moving the script tags down. It is usually convention to either put script tags just before the closing </body> tag or giving them an attribute of 'defer'.